### ISSUE TITLE
Add-in Error We couldn't connect to the catalog server for this add-in.

### ISSUE THREAD

**Issue by ndeeH** [OfficeDev/office-js#198]
Hi there,
we are getting this error:
**`
Add-in Error We couldn't connect to the catalog server for this add-in.
`**
when we switch our PowerPoint Content Add-In into Presentation Mode!

When we are NOT in presentation mode - our content add-in works as desired.
The manifest is deployed via O365 Admin Center...

_Any ideas why getting this? Please help - we are failing store validation because of this!_

TIA, Andi

**Comment by ndeeH** [comment]
To ensure it is not a problem in my add-in - I removed everything from my add-in.
Now I have only a plain HTML page and I also get this error in **presentation mode of PowerPoint Online**:

![image](https://user-images.githubusercontent.com/11518242/43125406-414b5e26-8f2a-11e8-80e6-889ce84f7293.png)

**_Any ideas?_**

**Comment by JuaneloJuanelo** [comment]
i wonder if you can provide a copy of your manifest so we can try it internally?
Also please share more details on what verision and Office builds and platforms you are experiencing this.

**Comment by ndeeH** [comment]
@JuaneloJuanelo
Hallo Juan, thank you for responding on my issues.
We are currently failing at store validation of our add-in and do not come any futher....

All issues have to do with the **online version of PowerPoint in presentation mode**.
I tested in this versions Chrome (67.0.3396.99) and also tried latest Firefox (61.0.1  64-Bit) - same problems.
To test the add-in with PPT Online we are using the Office 365 Admin Center to upload the [manifest](https://idlpresentation-qs.azurewebsites.net/Test/IDL.PRESENTATION.STORE.xml).

1. Upload the [Manifest](https://idlpresentation-qs.azurewebsites.net/Test/IDL.PRESENTATION.STORE.xml) to O365 via Admin Center
2. Open PPT Online (with O365 Account)
3.  Insert Add-In -> works
4. Switch to presentation mode:
![image](https://user-images.githubusercontent.com/11518242/43125024-291635a2-8f29-11e8-8f87-8aa9b6fe9247.png)

Another try was to directly upload the manifest into PPT Online (Office Add-In Dialog / Upload my add-in). There I get a different error in presentation mode:
![image](https://user-images.githubusercontent.com/11518242/43125159-79d608dc-8f29-11e8-8c6d-ae718ab97ef3.png)

We are seeing those errors in console which we are not able to associate with failures in our add-in:
```
Failed to load https://euc-powerpoint.officeapps.live.com/pods/RemoteUls.ashx?waccluster=NL1: 
The 'Access-Control-Allow-Origin' header contains multiple values 'https://contentstorage.osi.office.net, https://contentstorage.osi.office.net', 
but only one is allowed. Origin 'https://contentstorage.osi.office.net' is therefore not allowed access.
```
and
```
OSF_Outil_getQueryStringParamValue: The parameter is not found.
OSF_Outil_getQueryStringParamValue: The parameter is not found.
Error getting app data. Exception details: {}
Error getting app data. Exception details: {}
```

_Hope you could help us - our customers are waiting to use the store version of this add-in._

TIA, Andi

**Comment by JuaneloJuanelo** [comment]
@weird-dreams thanks for the detail, we'll take a look.

**Comment by ndeeH** [comment]
@JuaneloJuanelo Any news on this?

**Comment by ndeeH** [comment]
@JuaneloJuanelo Did your developers find anything why we are getting the error:
`Add-in Error We couldn't connect to the catalog server for this add-in.`

**Comment by JuaneloJuanelo** [comment]
Apparently there is a bug we are still investigating.

**Comment by ndeeH** [comment]
Hallo @JuaneloJuanelo , 
any news on this? For the current store validation our add in must work in PowerPoint Online in presentation mode. Here we are facing bugs which are not originated in our add in.
We could show a page `"Add in not supported in PowerPoint Online"` - but I think this will not pass validation. (There is no requirement set that will hide the add in in the online versions)
Any ideas?
TIA, Andi

**Comment by ndeeH** [comment]
Hallo @JuaneloJuanelo ,
this issue is still present just tested again in PowerPoint Online in newest Chrome Version.
When using the manifest earlier we get the error:
`We couldn't connect to the catalog server for this add-in.`

I have also seen a **new bug** - this might have to do with my thumbnail issue #184 
When scaling content add ins there seems to be a overlay bitmap AND the content that scales differently:
![image](https://user-images.githubusercontent.com/11518242/45366812-02424780-b5e0-11e8-8c01-f90fe67d41fe.png)
This also happens with many content add ins which are already in the store.

_Please let us know, when we could test again.
I will try to get our add in through store verification and mention those open bugs here....
(Are you in contact with the verification team?)_

TIA, Andi

**Comment by ndeeH** [comment]
@JuaneloJuanelo Unfortunately - I tested with a wrong manifest - this issue is still present. :(

With the [manifest of our QA version](https://idlpresentation-qs.azurewebsites.net/Test/IDL.PRESENTATION.QS.xml) this issue has gone. 
The add ins and azure settings are identical. Only the deployment goes to a different azure subscription.

**Comment by LiXiaoooowei** [comment]
@weird-dreams I recently developed a content add-in for PowerPoint and faced the same wierd issue with PowerPoint online version. May I know which azure subscription did you use to fix the issue?

**Comment by ndeeH** [comment]
@LiXiaoooowei 
I tested again and it seems our add in is set to **work only in the desktop versions**.
It seems MS has a switch to do this, because we do not have any requirement restrictions to do so.
@JuaneloJuanelo The error still exists! May the store verification set to only support the desktop versions?

When we create our content add in the desktop version and open it via OneDrive in PPT online - then we get these notifications (always clicking on first button):
![image](https://user-images.githubusercontent.com/11518242/47737486-5200cf00-dc71-11e8-8f48-dc65b3122924.png)

![image](https://user-images.githubusercontent.com/11518242/47737530-69d85300-dc71-11e8-92e3-5ac94d9fadb5.png)

![image](https://user-images.githubusercontent.com/11518242/47737562-7c528c80-dc71-11e8-8a3a-a610148fb0ab.png)

**Comment by kbrandl** [comment]
@JuaneloJuanelo are you able to provide any update on this issue?  FYI, seemingly related issue #302 contains steps to repro this issue.

**Comment by nirorlev** [comment]
@JuaneloJuanelo We're also experiencing the same issue on a different add-in that we develop.
If you need another add-in code for debugging this - I can share ours.
![image](https://user-images.githubusercontent.com/2757372/49741116-6091cb80-fc9e-11e8-97b6-83d2df600a4b.png)

**Comment by ndeeH** [comment]
@nirorlev Your error message seams different from mine "_..couldn't connect to the catalog server..._". 
Is there an exception outside of your code which leads to this error when switching to presentation mode?

**Comment by JuaneloJuanelo** [comment]
guys i was able to repro this issue, it seems to only be happening when side loading add-ins and also when using corp catalog.

**Comment by JuaneloJuanelo** [comment]
we are tracking with 2737719  we'll fix it ASAP thanks all for your feedback

**Comment by JuaneloJuanelo** [comment]
closing as a dupe of https://github.com/OfficeDev/office-js/issues/302

**Comment by Gq77** [comment]
Same issue with a new iMac here, using office 365 for mac.  Cannot connect to Store... any resolution? Microsoft mentions this issue but not other solution - sign out and sign in? STILL not working get add-ins

**Comment by AlexJerabek** [comment]
Hi @Gq77,

I'm sorry to hear you're having issues with the platform. Given the age of this issue, could you create a new issue and provide your add-in's manifest, the steps to reproduce the error, any screenshots of the error? That would help us with the specifics of your problem.

Thank you

**Comment by Gq77** [comment]
Hi Mr. AlexJerabek,
Here is the issue with my office 365 on Mac (screen shot below) I have 3 email accounts for my work; one of those is a
IMAP/POP account.  I understand that it could affect somehow.  But my other accounts are MS exchange.  Please help, I am
a bit confused.  Office 365 version is updated to : 16.53.21091200 as of sep 14 2021
Thanks
![Snip20210923_2](https://user-images.githubusercontent.com/61130609/134520131-ad4c1941-7a0f-457d-bc97-99b07055f0e2.png)

**Comment by AlexJerabek** [comment]
Hi @Gq77,
Are you able to connect to the Office Store from your Office 365 apps on Windows or on the web? The message you're seeing is caused by [admin settings](https://answers.microsoft.com/en-us/msoffice/forum/all/office-add-ins-prevented-by-365/e01f07df-618e-4ce7-acf3-cde0973f04da), so we need to verify the behavior is different on Mac versus Windows/Web.

**Comment by Gq77** [comment]
Mr. Alex, I checked our IT admin settings.  We are not blocked by our IT.  We are a non profit agency, our offc 365 is given to our computers logging thru our office account with a password.  My only conclusion is... because our email here in is an IMAP/POP anticquated type email - not an exchange account;  maybe we are not connecting to the Office Store?  Is all I can think of.

**Comment by AlexJerabek** [comment]
@exextoc, could you please investigate @Gq77's latest issues?

**Comment by Gq77** [comment]
Hi @AlexJerabek,  in Office 365 -2016 -2019 MAC/WINDOWS both the Office Store won't open or interact with any suscription if the emails in use are not exchange or outlook based emails.  I just tested this issue.  Adding a personal email from Live.com.  When added, the Office Store immediately opened, in Word Excel and Power Point Of course, case in point.  This email is my personal email, cannot use for work ... so tested.   In hopes this helps others going thru the same issue. 

Another conclusion, exchg based email accounts OST. do not equal PST.  files we can keep in our system.   To me that is another downside of the equation.  "I keep receipts"

**Comment by Oleg-O** [comment]
Correct, as [this doc page](https://learn.microsoft.com/en-us/office/dev/add-ins/outlook/add-in-requirements) states, "If the client is configured to connect with POP3 or IMAP, add-ins will not load.".

**Comment by Tannu17** [comment]
Trying to auto launch my word addin but getting the below error
We can't load this add-in because we couldn't connect to the catalog
 
can you please check. If there is some permission issue or not?

![ss](https://github.com/user-attachments/assets/d5532e1f-acdb-47a1-92e6-ef294cf60ee5)
